zokrates_fs_resolver/Cargo.toml

```diff
@@ -3,13 +3,10 @@ name = "zokrates_fs_resolver"
 version = "0.5.0"
 authors = ["Thibaut Schaeffer <thibaut@schaeff.fr>"]
 repository = "https://github.com/JacobEberhardt/ZoKrates.git"
-
-[features]
-default = []
+edition = "2018"
 
 [dependencies]
+zokrates_core = { version = "0.4.2", path = "../zokrates_core" }
```

Comment:
Is there a way to keep `fs_resolver` independent from `zokrates_core`? I guess having a different crate for that trait definition?

Reply:
We could introduce a "common" crate that contains the trait definition, and the trait should be more generic then.